.github/workflows/main.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        tox_env: [py39, py310,  py311,  py312,  py313, pypy3, pep8]
```

Comment:
TIL: tox apparently supports default presets which would work if `pep8` was installed in the workflow or my local system :)